src/classes/astandardpaths.cpp

```diff
@@ -1,47 +1,47 @@
 #include "src/classes/astandardpaths.h"
 
 //QMap<QStandardPaths::StandardLocation, QString> AStandardPaths::paths;
-QMap<AStandardPaths::Dirs, QString> AStandardPaths::paths;
+QMap<AStandardPaths::Dirs, QString> AStandardPaths::dirs;
 
 void AStandardPaths::setup()
 {
     auto settings_location = QStandardPaths::AppConfigLocation;
     auto data_location = QStandardPaths::AppDataLocation;
-    paths = {  // [G]: potential rename to `dirs`
+    dirs = {  // [G]: potential rename to `dirs`
         {Database, QStandardPaths::writableLocation(data_location)},
         {Templates, QDir(QStandardPaths::writableLocation(data_location)).filePath("templates")},
         {Settings, QStandardPaths::writableLocation(settings_location)},
```

Comment:
I think this is obsolete since we are using the default constructor of QSettings now. This results in the ini file being created in 
`~/.config/opl/openPilotLog.ini`, so creating an empty folder `openPilotLog` in the same directory where `openPilotLog.ini` is located looks dirty.
![tree](https://user-images.githubusercontent.com/32618887/103483756-c7d0ca00-4de9-11eb-826c-66d1c5f11f62.png)

Reply:
So the default constructor doesnt conform to AppConfigLocation?

Reply:
it does use standard [locations](https://doc.qt.io/qt-5/qsettings.html#locations-where-application-settings-are-stored), depending on the [scope](https://doc.qt.io/qt-5/qsettings.html#setPath) but the path does only include the orgname, the appname is used for the settings file itself:
 `$HOME/.config/opl/openPilotLog.ini`
The path is automatically created if needed so we don't need to manually handle it.

Reply:
so we are ok QSettings handles its as it should even if it seems off in our eyes.
Any custom configs will go to opl/openPilotLog wouldnt it?

Reply:
yes, `opl/openPilotLog.ini`